src/messages/layouts/MessageLayout.cpp

```diff
@@ -443,12 +443,31 @@ void MessageLayout::deleteCache()
 // returns nullptr if none was found
 
 // fourtf: this should return a MessageLayoutItem
-const MessageLayoutElement *MessageLayout::getElementAt(QPoint point)
+const MessageLayoutElement *MessageLayout::getElementAt(QPoint point) const
 {
     // go through all words and return the first one that contains the point.
     return this->container_.getElementAt(point);
 }
 
+std::pair<int, int> MessageLayout::getWordBounds(QPoint relativePos) const
+{
+    const auto *element = this->getElementAt(relativePos);
+    assert(element != nullptr);
+
+    if (element->getWordId() != -1)
+    {
+        return this->container_.getWordBounds(element);
+    }
```

Comment:
Could you add a comment here for what this significes? It's not clear to me what it means when the element we're looking at has a word id of -1

Reply:
-1 basically means that the element can't span across multiple lines (like an emote, for example). I agree the code is not very self explanatory here. Do you want a comment here or should I encapsulate that inside a function like `element->canBeMultiline()` (name is kinda bad but you get the idea)

Reply:
I think a comment here is fine

Reply:
Added